python/lsst/rubintv/production/slac/botTesting.py

```diff
@@ -119,16 +121,46 @@ def getNumExpectedItems(instrument, expRecord):
     expRecord : `lsst.daf.butler.DimensionRecord`
         The exposure record. This is currently unused, but will be used once
         we are doing this properly.
+    logger : `logging.Logger`
+        The logger, created if not supplied.
     """
+    if logger is None:
+        logger = logging.getLogger(__name__)
+
+    fallbackValue = None
     if instrument == "LATISS":
-        return 1
+        fallbackValue = 1
     elif instrument == "LSSTCam":
-        return 201
+        fallbackValue = 201
     elif instrument in ["LSST-TS8", "LSSTComCam"]:
-        return 9
+        fallbackValue = 9
     else:
         raise ValueError(f"Unknown instrument {instrument}")
 
+    try:
+        resourcePath = (f"s3://rubin-sts/{expRecord.instrument}/{expRecord.day_obs}/{expRecord.obs_id}/"
+                        f"{expRecord.obs_id}_expectedSensors.json")
+        url = ResourcePath(resourcePath)
+        jsonData = url.read()
+        data = json.loads(jsonData)
```

Comment:
That is correct - these are supplementary files from the camera so that you know what to expect/ingest. If everything ingests correctly then at t=2-3s then the butler will contain the same info, but this provides a way of knowing what to wait for.